### PR TITLE
Made changes to create-index documentation to add copy-as-curl option

### PR DESCRIPTION
**Pull request by niyatiagg**
### Description
Incorporated 1 click copy-as-curl option for create-index documentation

### Issues Resolved
Closes [#6823]

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/index-apis/delete-index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ DELETE /sample-index
 {% include copy-curl.html %}
 
 ## Path and HTTP methods
-
+****
```
Comment (Naarcha-AWS) [inline _api-reference/index-apis/delete-index.md:24]:
```suggestion
Do we need these here? 
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _api-reference/index-apis/delete-index.md (hunk 1 above)
Comment (niyatiagg) [reply _api-reference/index-apis/delete-index.md:24]: You are right. Its not!

File: _api-reference/index-apis/delete-index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ DELETE /sample-index
 {% include copy-curl.html %}
 
 ## Path and HTTP methods
-
+Do we need these here?
```
Comment (kolchfa-aws) [inline _api-reference/index-apis/delete-index.md:24]:
```suggestion

```

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**